From 783351fb3038cb5b339175884a6f74a35712d8c0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Felix=20Paul=20K=C3=BChne?= <fkuehne@videolan.org>
Date: Wed, 22 Jan 2014 13:37:59 +0100
Subject: [PATCH 07/22] contrib/speex: fix compilation for non-ARMv7 iOS
 targets

the neon code included in speex doesn't do ARMv8 yet
---
 contrib/src/speex/rules.mak | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/contrib/src/speex/rules.mak b/contrib/src/speex/rules.mak
index f9ff8d9..cd1794a 100644
--- a/contrib/src/speex/rules.mak
+++ b/contrib/src/speex/rules.mak
@@ -34,6 +34,11 @@ ifeq ($(ARCH),arm)
 SPEEX_CONF += --enable-arm5e-asm
 endif
 endif
+ifdef HAVE_IOS
+ifndef HAVE_ARMV7A
+SPEEX_CONF += --disable-neon
+endif
+endif
 
 .speex: speex
 	mkdir -p $</m4 && $(RECONF)
-- 
1.9.3 (Apple Git-50)

